### PR TITLE
Feature signup login page

### PR DESCRIPTION
**Pull request by OsmanRana**
### What this PR does (required):
- Changed the login/logout button and the button text as per mock up.
- Made Title bold.
- Form TextField variant="standard" added.

### Screenshots / Videos (front-end only):
- 
login-page

### Any information needed to test this feature (required):
- Post what steps to take to be able to test this feature (eg. "sign up as a new user and upload a profile picture from the navbar")

### Any issues with the current functionality (optional):
- List any problems where you were not able to complete all tasks on ticket
- List any problems this PR may cause for the project or other active PRs
- Post a screenshot/video of the problem, along with detailed console outputs where applicable

### CODE REVIEW

**Review by rajivtitus — COMMENTED** [review]
Good start! Please match the UI to the mocks given

**Review by rajivtitus — CHANGES_REQUESTED** [review]
Please complete this feature

**Comment by OsmanRana** [comment]
### What this PR does (required):
Edited signup and login page

### Screenshots / Videos (front-end only):
signup
login

File: client/src/pages/Login/LoginForm/LoginForm.tsx [hunk 1]
```diff
@@ -46,50 +48,79 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
...
+              type="submit"
+              size="large"
+              className={classes.submit}
+              sx={{
+                backgroundColor: '#f14140',
+                color: 'white',
+                '&:hover': {
+                  backgroundColor: 'white',
+                  color: '#f14140',
+                  border: '1px solid #f14140',
```
Comment (moffatethan) [inline client/src/pages/Login/LoginForm/LoginForm.tsx:114]: I think just the default MUI hover state is fine and less of an big color change.

File: client/src/pages/Login/LoginForm/LoginForm.tsx [hunk 2]
```diff
@@ -46,50 +48,79 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
...
+              }}
+              name="email"
+              autoComplete="email"
+              autoFocus
+              helperText={touched.email ? errors.email : ''}
+              error={touched.email && Boolean(errors.email)}
+              value={values.email}
+              onChange={handleChange}
+            />
+            <InputLabel className={classes.inputLabel}>
```
Comment (moffatethan) [inline client/src/pages/Login/LoginForm/LoginForm.tsx:77]: Probably could've made this a reusable component.

**Review by moffatethan — CHANGES_REQUESTED** [review]
Just some minor changes, otherwise pretty good.

**Pull request merged**